src/pages/assetGroup/AssetGroup.jsx

```diff
@@ -22,10 +22,27 @@ import MoreMenu from '../../components/MoreMenu';
 import ConfirmDelete from '../../components/ConfirmDelete';
 import EntityHeader from '../../components/EntityHeader';
 import CustomAlert from '../../components/Alert';
+import ProgressMetrics from '../../components/progress/ProgressMetrics';
 import AGSTable from './AGSTable';
 
 const POLLING_INTERVAL = 5000; // 5 seconds
 
+const useStyles = makeStyles(theme => ({
+  alert: {
+    // extend the progress bar the full width of the alert
+    '& .MuiAlert-message': {
+      flexGrow: 1,
+    },
+    // use the info alert's colors instead of the primary site color
+    '& .MuiLinearProgress-colorPrimary': {
+      backgroundColor: lighten(theme.palette.info.light, 0.5),
+    },
+    '& .MuiLinearProgress-barColorPrimary': {
+      backgroundColor: theme.palette.info.main,
+    },
+  },
+}));
```

Comment:
I know that we try to avoid `makeStyles`, but it was the best way that I found to access these elements.